### ISSUE TITLE
Support for air conditioning control

### ISSUE THREAD

**Issue by markcamilleri** [WouterEekhout/MMM-Tado#6]
here's my output:

`{
	"tadoMode": "AWAY",
	"geolocationOverride": false,
	"geolocationOverrideDisableTime": null,
	"preparation": null,
	"setting": {
		"type": "AIR_CONDITIONING",
		"power": "OFF"
	},
	"overlayType": null,
	"overlay": null,
	"openWindow": null,
	"nextScheduleChange": null,
	"nextTimeBlock": {
		"start": "2019-04-17T20:00:00.000Z"
	},
	"link": {
		"state": "ONLINE"
	},
	"activityDataPoints": {
		"acPower": {
			"timestamp": "2019-04-16T17:02:37.895Z",
			"type": "POWER",
			"value": "OFF"
		}
	},
	"sensorDataPoints": {
		"insideTemperature": {
			"celsius": 18.45,
			"fahrenheit": 65.21,
			"timestamp": "2019-04-17T09:03:11.316Z",
			"type": "TEMPERATURE",
			"precision": {
				"celsius": 0.1,
				"fahrenheit": 0.1
			}
		},
		"humidity": {
			"type": "PERCENTAGE",
			"percentage": 76.90,
			"timestamp": "2019-04-17T09:03:11.316Z"
		}
	}
}
`

**Comment by WouterEekhout** [comment]
@markcamilleri I'm sorry to hear that. Does it show the Tado logo or nothing at all?

**Comment by markcamilleri** [comment]
@WouterEekhout It does show the logo in fact, but nothing else

**Comment by WouterEekhout** [comment]
@markcamilleri thank your for the output it is useful. It seems in the section setting you are only using the air conditioning control of Tado. Is this correct? I didn't build this in because I do not have access to one. I am willing to help built it in, but I will need your help. I need your output when the air conditioning is on and you need to test it for me. Is this oke?

```JavaScript
	"setting": {
		"type": "AIR_CONDITIONING",
		"power": "OFF"
	},
```

**Comment by markcamilleri** [comment]
@WouterEekhout Thanks for your reply. I am sending you the settings below

`{
	"tadoMode": "AWAY",
	"geolocationOverride": false,
	"geolocationOverrideDisableTime": null,
	"preparation": null,
	"setting": {
		"type": "AIR_CONDITIONING",
		"power": "ON",
		"mode": "HEAT",
		"temperature": {
			"celsius": 24.00,
			"fahrenheit": 75.20
		}
	},
	"overlayType": "MANUAL",
	"overlay": {
		"type": "MANUAL",
		"setting": {
			"type": "AIR_CONDITIONING",
			"power": "ON",
			"mode": "HEAT",
			"temperature": {
				"celsius": 24.00,
				"fahrenheit": 75.20
			}
		},
		"termination": {
			"type": "MANUAL",
			"typeSkillBasedApp": "MANUAL",
			"projectedExpiry": null
		}
	},
	"openWindow": null,
	"nextScheduleChange": null,
	"nextTimeBlock": {
		"start": "2019-04-18T20:00:00.000Z"
	},
	"link": {
		"state": "ONLINE"
	},
	"activityDataPoints": {
		"acPower": {
			"timestamp": "2019-04-18T06:14:07.214Z",
			"type": "POWER",
			"value": "ON"
		}
	},
	"sensorDataPoints": {
		"insideTemperature": {
			"celsius": 18.59,
			"fahrenheit": 65.46,
			"timestamp": "2019-04-18T06:02:45.710Z",
			"type": "TEMPERATURE",
			"precision": {
				"celsius": 0.1,
				"fahrenheit": 0.1
			}
		},
		"humidity": {
			"type": "PERCENTAGE",
			"percentage": 79.40,
			"timestamp": "2019-04-18T06:02:45.710Z"
		}
	}
}
`

**Comment by WouterEekhout** [comment]
@markcamilleri great! Thanks! I will keep you posted.

**Comment by WouterEekhout** [comment]
@markcamilleri small question. In last output the setting is as followed:
```JavaScript
"setting": {
		"type": "AIR_CONDITIONING",
		"power": "ON",
		"mode": "HEAT",
		"temperature": {
			"celsius": 24.00,
			"fahrenheit": 75.20
		}
	},
```
It contains de mode option and it states "HEAT". If it is cooling down, does it states "COOL" or ...? I will assume COOL for now.

**Comment by WouterEekhout** [comment]
@markcamilleri for this feature I've created a feature branch: air_conditioning-support. If everything works I will merge it into the master branch. I've already made code changes. Everything should work at the moment. Next to a heating icon if it is heating it will also show a snowflake if it is cooling. 

Execute the following commands to be on the correct branch, restart the mirror and let me know if it works.
```
git fetch
git checkout air_conditioning-support
```
